src/logic/garden-logic.js

```diff
@@ -16,13 +24,23 @@ export default function useAppLogic() {
 
   return {
     actions,
-    commonPool,
+    commonPool: { value: commonPool, token: config?.conviction.requestToken },
     config,
     errors,
     filters,
     loading: loading || !blockHasLoaded,
-    mainToken,
+    priceToken: mainToken.data,
     proposals,
     proposalsFetchedCount,
+    totalActiveTokens: {
+      value: config?.conviction.totalStaked,
+      token: config?.conviction.stakeToken,
+    },
+    // mainToken: BYOT ? wrappableToken : token
```

Comment:
Let's remove this line.

Reply:
Ah i put this line not as something commented that forgot to remove but for others to consider what the main token is. Perhaps we can change it to plain english ? lol